content/docs/dvclive/get-started.md

```diff
@@ -3,164 +3,189 @@
-
-```python
-# train.py
+<admon type="info">
 
-from dvclive import Live
+Note that the path passed to the `--live` option (`"dvclive"`) matches the
+`path` passed to [`Live()`](/doc/dvclive/api-reference/live) in the Python code
+(`"dvclive"` is used as default when no value is passed).
 
-live = Live()
+</admon>
 
-live.log_param(NUM_EPOCHS)
+```dvc
+$ dvc exp init \
+--live "dvclive" \
+--code "train.py" \
+python train.py
+```
 
-for epoch in range(NUM_EPOCHS):
-    train_model(...)
-    metrics = evaluate_model(...)
+<details>
+
+### ⚙️ Expand to see the `dvc.yaml`
+
+The `dvc.yaml` will contain a new `train` stage using the DVCLive outputs as
+`metrics` and `plots`:
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - train.py
+    metrics:
+      - dvclive.json:
+          cache: false
+    plots:
+      - dvclive/scalars:
+          cache: false
+```
 
-    for metric_name, value in metrics.items():
-        live.log(metric_name, value)
+If you log other types of data you can check the
+[Output folder structure](/doc/dvclive/outputs) and update your `dvc.yaml`
+accordingly.
 
-    live.next_step()
-```
+</details>
 
-## Outputs
+## Run experiments
 
-After you run your training code, you should see the following content in the
-project:
+You can now
+[run experiments](/doc/user-guide/experiment-management/running-experiments)
+with `dvc exp run`:
 
 ```dvc
-$ tree
-├── dvclive
-│   ├── images
-│   │   └── img.png
-│   ├── params.yaml
-│   ├── plots
-│   │   └── roc.json
-│   └── scalars
-│       └── acc.tsv
-└── dvclive.json
+$ dvc exp run
 ```
 
-The contents of the `dvclive` folder and `dvclive.json` will vary depending on
-the type of data you have logged and whether you have updated the step value or
-not.
+## Compare and visualize experiments
 
-See `Live.log()`, `Live.log_param()`/`Live.log_params()`, `Live.log_image()` and
-`Live.log_plot()` for more details.
+After following the above steps, you have enabled different ways of monitoring
+the training, comparing, and visualizing the experiments:
 
-### Metrics report
+### DVC
 
-If and when `step` is updated, DVCLive generates or updates a report in
-`dvclive/report.{format}` which will contain all the logged data.
+The [metrics history](/doc/dvclive/api-reference/live/log#step-updates)
+`dvclive/scalars` can be visualized with `dvc plots`.
 
-![](/img/dvclive-html.gif)
+The [metrics summary](/doc/dvclive/api-reference/live/log#description) in
+`dvclive.json` can be visualized with `dvc metrics`.
 
-The `format` can be HTML) or Markdown depending on the value of the `report`
-argument passed to [`Live()`](/doc/dvclive/api-reference/live#parameters).
+If you have logged parameters with `Live.log_param()`, they will be stored in
+`dvclive/params.yaml` and can be visualized with `dvc params`.
 
-<admon type="info">
+In addition, the `dvc exp show` and `dvc exp diff` commands can be used to
+compare both metrics and parameters across experiments.
 
```

Comment:
I don't think it's worth going through every related DVC command here. Mentioning `dvc exp show` and `dvc plots show/diff` is probably enough.